### PR TITLE
Only set CREATED_PDR group if FTUP was enabled

### PR DESCRIPTION
**Pull request by sergeymatov**
Also revert freeing of created_pdr vec for response on session
modification/establishment request.

### CODE REVIEW

**Comment by ivan4th** [comment]
LGTM

File: upf/upf_pfcp_api.c [hunk 1]
```diff
@@ -728,7 +728,6 @@ handle_f_teid (upf_session_t * sx, upf_main_t * gtm, pfcp_pdi_t * pdi,
 	{
 	  vec_add2 (*created_pdr_vec, created_pdr, 1);
 	  memset (created_pdr, 0, sizeof (*created_pdr));
-	  SET_BIT (grp->fields, SESSION_ESTABLISHMENT_RESPONSE_CREATED_PDR);
```
Comment (RoadRunnr) [inline upf/upf_pfcp_api.c:731]: This removes the only user of the `grp` function argument. The argument should also be removed.

**Review by RoadRunnr — CHANGES_REQUESTED** [review]
While reviewing this change, I also discovered that handling of the `create` argument to handle_f_teid is broken. Will open a seperate issue.

File: upf/upf_pfcp_api.c (hunk 1 above)
Comment (sergeymatov) [reply upf/upf_pfcp_api.c:731]: fixed, thanks for note

**Review by RoadRunnr — APPROVED** [review]
LGTM

**Comment by ivan4th** [comment]
LGTM again :)

**Pull request merged**